shared/constants/swaps.ts

```diff
@@ -158,6 +165,7 @@ export const ALLOWED_PROD_SWAPS_CHAIN_IDS = [
   CHAIN_IDS.AVALANCHE,
   CHAIN_IDS.OPTIMISM,
   CHAIN_IDS.ARBITRUM,
+  CHAIN_IDS.ZKSYNC,
```

Comment:
Better to name ZKSYNC_ERA

Reply:
Great suggestion, I've renamed other variables as well.